cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -64,6 +64,8 @@ API Changes
 Known Issues
 ------------
 
+- CDAP has been tested on and supports CDH 4.2.x through CDH 5.3.x. It has not been tested on later versions.
```

Comment:
And, while we are at, what about Hadoop? 2.7.0 is available, but what about us (for this and the following)?

"Zookeeper 3.4.3"; "3.4.6" its current "stable" release.

"Hive 12.0" -> should probably be written "Hive 0.12.0". Latest Hive is "1.1.0".